spec/requests/home_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+RSpec.describe 'Fizz Buzz Method' do
+    
+    # When a valid number is passed
+    describe 'GET buzify' do
+        before { get "/buzify?number=#{number}" }
+
+        context 'when a valid number passed' do
+
+            let(:number) { 1 }
+
+            it 'returns status code 200' do
```

Comment:
here you can just stub service call for example in the before `allow(FizzBuzzService).to receive(:call).with(1).and_return('')` or `allow(FizzBuzzService).to receive(:call).with(15).and_return('FizzBuzz')` that way you don't have to really test for the business functionality but rather whatever the service comes up with, I know which response code and response body to serve to user.